gulpfile.js/tasks/browserify.js

```diff
@@ -38,6 +38,11 @@ var browserifyTask = function (callback, devMode) {
       bundleConfig = _.omit(bundleConfig, ['external', 'require'])
     }
 
+    if (!isDev) {
+      // stop absolute urls appearing in prod browserify'd bundles
+      _.extend(bundleConfig, {fullPaths: false, plugin: collapse})
```

Comment:
Yes `fullPaths: false` doesn't actually show any changes in the diff (diff with and without the option) but as mentioned in the PR the docs say [it should](https://github.com/substack/node-browserify#usage). I also bumped our version of `Browserify` (currently declared via [karma-browserify](https://github.com/hmrc/assets-frontend/blob/master/package.json#L63)) in `package.json` to `v.14.1.0` to see if that made this option work but it didn't.